translations/ui-circulation/en.json

```diff
@@ -465,5 +465,7 @@
   "permission.settings.other-settings": "Settings (Circ): Can create, edit and remove other settings",
   "permission.settings.notice-templates": "Settings (Circ): Can create, edit and remove patron notice templates",
   "permission.settings.overdue-fines-policies": "Settings (Circ): Can create, edit and remove overdue fine policies",
-  "permission.settings.lost-item-fees-policies": "Settings (Circ): Can create, edit and remove lost item fee policies"
+  "permission.settings.lost-item-fees-policies": "Settings (Circ): Can create, edit and remove lost item fee policies",
+
+  "mod-circulation.noItemFoundForBarcode": "No item with barcode {itemBarcode} exists"
```

Comment:
@julianladisch Could you please add screenshot that showed that current translation key are used in module or we need to add changes in our module for use current translation key?

Reply:
We just need to add the translation key.
All other tasks are out of scope and have separate Jiras:
* https://issues.folio.org/browse/UIREQ-603 "Internationalize back-end failure messages"
* https://issues.folio.org/browse/CIRC-1146 "Internationalize back-end failure messages"
  = https://github.com/folio-org/mod-circulation/pull/879/files